pkg/assembler/graphql/schema/hasSBOM.graphql

```diff
@@ -116,9 +116,9 @@ type HasSBOMEdge {
 }
 
 extend type Query {
-  "Returns all SBOM certifications."
+  "Returns all SBOM certifications (by default it filters out all deps.dev hasSBOMs)."
```

Comment:
How do i run a query if i want to include deps.dev results? i wonder if this is something we want to bake in on the server side or just have it be a default client query? Like HasSbomOpiniated query?

I guess we would have to add the schema change where HasSBOMSpec has an exclude collectors?

Reply:
so if you specify the hasSBOM spec collect = deps.dev, it will include deps.dev SBOM otherwise they will be ignored

Reply:
As this point, where we removed deps.dev collector long term, they would never be used in the query.

Reply:
> I guess we would have to add the schema change where HasSBOMSpec has an exclude collectors?

I don't think we want to bake in `not equal` into graphql..

Reply:
@lumjjb I removed the HasSbom opinionated filtering on deps.dev! PTAL